apps/network-alerts/js/notification/notification.js

```diff
@@ -18,12 +18,12 @@ function onNotification(message) {
     return;
   }
 
-  var title = Utils.parseParams(message.imageURL).titleID;
+  var title = message.data.title;
```

Comment:
__nit:__ if you don't use ```Utils``` in this file you won't need jshint directive as well ``` /* global Utils */``` (see top of the file) - same in tests. Moreover you don't need it in ```notification.html``` - clear performance win!